src/carvel/config/dataflow-deployment.yml

```diff
@@ -43,16 +43,24 @@ spec:
           readOnly: true
         ports:
         - containerPort: 80
+        startupProbe:
+          httpGet:
+            port: 9393
+            path: #@ dataflow_liveness_path()
```

Comment:
Should we have a `dataflow_startup-path()`?

Reply:
The liveness probe is a good indication that startup is complete.